test/functional/autocmd/vvar_event_spec.lua

```diff
@@ -0,0 +1,15 @@
+local helpers = require('test.functional.helpers')
```

Comment:
This test file should probably live in `eval/`:

    test/functional/eval/vvar_event_spec.lua

(or `test/viml` -- I don't know why we have both...)